src/test/java/TC_DP194_49.java

```diff
@@ -0,0 +1,27 @@
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+import steps.CategoryPageStep;
+import steps.CheckoutPageAsGuestStep;
+import steps.HomePageStep;
+
+public class TC_DP194_49 extends CommonConditionTest{
+
+    @ParameterizedTest
+    @CsvFileSource(resources = "/GuestValuesCheckoutPage.csv",numLinesToSkip = 1)
+    public void checkoutAsGuest(String name, String surname, String email, String phone, String address, String city, String postCode, int countryIndex, String regionIndex){
+        String contactPageUrl = "http://34.121.117.87/index.php?route=checkout/success";
```

Comment:
We use as an expected URL constants in PageURL interface (constants package). You should have it (package) if you pulled last updates from div branch